### ISSUE TITLE
Optimizer Hyperparameter for MoleculeNet

### ISSUE THREAD

**Issue by chao1224** [deepchem/deepchem#1256]
Hi, I want to rerun some models reported from MoleculeNet.
I know the models are stored at [http://deepchem.io.s3-website-us-west-1.amazonaws.com/trained_models/Hyperparameter_MoleculeNetv3.tar.gz](http://deepchem.io.s3-website-us-west-1.amazonaws.com/trained_models/Hyperparameter_MoleculeNetv3.tar.gz), but can we also find other optimizer hyperparameters like learning rate, epoch number, batch size?

In the `model_validation` folder, I found some python scripts on some models, but not all of them. Are these sciprts used for the actual training?

**Comment by peastman** [comment]
I believe the hyperparameters used are the ones in https://github.com/deepchem/deepchem/blob/master/deepchem/molnet/preset_hyper_parameters.py.  Is that correct?

**Comment by chao1224** [comment]
@peastman Thanks. 
If so, this implies all models are using same hyperparameter when switching among different tasks right?

**Comment by lilleswing** [comment]
The final hyperparameters are here
Hyperparameters and all benchmark scripts for MoleculeNet are available at:
http://deepchem.io.s3-website-us-west-1.amazonaws.com/trained_models/Hyperparameter_MoleculeNetv3.tar.gz

They are stored as python 2.7 pickle files.

**Comment by miaecle** [comment]
@chao1224 Sorry for the confusion. @lilleswing is correct, we provided pickle files which stored dict of hyper parameters, which should include learning rate and batch size in most of them. `model_validation` folder provided codes and saved models that are used to compare our implementations with the original articles that proposed these models, details please refer to the appendix(SI) of MoleculeNet.

**Comment by chao1224** [comment]
I see. So the files correspond to the hyperparameters, not model weights.
Thanks. @lilleswing @miaecle
